scripts/rebase-ledger.js

```diff
@@ -0,0 +1,48 @@
+const fs = require("fs");
+const sc = require("../node_modules/sourcecred/dist/server/api.js").sourcecred;
+const isEqual = require("../node_modules/lodash.isequal");
+
+const rawToLedgerLog = (rawString) =>
+  JSON.parse("[" + rawString.trim().split("\n").join(",") + "]");
```

Comment:
Seems rather hacky, and given we're talking megabytes of data here fairly inefficient.

Instead of reinventing the parsing here, use sourcecred's own parse feature?
https://github.com/sourcecred/sourcecred/blob/bcdb96822b5393898fc7763bb3e0940052b201cd/packages/sourcecred/src/core/ledger/ledger.js#L887

This would also include all the latest and greatest "legacy workaround fixes" and other non-standard stuff you might encounter in these ndjson-like files.

Reply:
permalink for readability: https://github.com/sourcecred/sourcecred/blob/bcdb96822b5393898fc7763bb3e0940052b201cd/packages/sourcecred/src/core/ledger/ledger.js#L887

Reply:
of course, silly me